src/cat.rs

```diff
@@ -154,41 +151,30 @@ fn show_locals<W: io::Write>(locals: &[String], _opts: &Options, output: &mut W)
         .filter(|&name| {
             name != "globals.local" && name != "pre-globals.local" && name != "post-globals.local"
         })
-        .filter_map(|name| match find_profile(name) {
-            Some(path) => Some(path),
-            None => {
-                warn!("{} could not be found.", name);
-                None
-            }
+        .filter_map(|name| {
+            let flags = ProfileFlags::default();
+            let profile = Profile::new(name, flags).unwrap();
+            Some(profile)
```

Comment:
Can be `map` if it always returns `Some(_)`.
```suggestion
        .map(|name| {
            let flags = ProfileFlags::default();
            Profile::new(name, flags).unwrap()
```

Reply:
Noted.